pkg/apis/kubernetes/v1/environments.go

```diff
@@ -0,0 +1,14 @@
+package v1
+
+import (
+	"go.anx.io/go-anxcloud/pkg/api"
+	"go.anx.io/go-anxcloud/pkg/api/types"
+)
+
+func EnvironmentProduction(override bool) types.Option {
```

Comment:
missing doc comment

Reply:
Added the comment (not pushed yet).

But do we need this file and its content at all in this project? We are already defaulting to the prod environment.

Reply:
hm probably not, good point - up to you